solutions/secure-regional-bucket/outputs.tf

```diff
@@ -5,3 +5,23 @@ output "buckets" {
   description = "List of buckets created"
   value       = module.cos.buckets
 }
+
+output "s3_endpoint_direct" {
+  description = "The s3 direct endpoint of the created bucket."
+  value       = try(module.cos.buckets[var.bucket_name].s3_endpoint_direct, "")
+}
+
+output "s3_endpoint_private" {
+  description = "The s3 private endpoint of the created bucket."
+  value       = try(module.cos.buckets[var.bucket_name].s3_endpoint_private, "")
+}
+
+output "bucket_name" {
+  description = "The name of the bucket that was created. Includes the optional suffix if enabled."
+  value       = try(module.cos.buckets[var.bucket_name].bucket_name, "")
+}
+
+output "cos_instance_crn" {
```

Comment:
perhaps call the output `cos_instance_id` so it is consistent with the input (`existing_cos_instance_id`)